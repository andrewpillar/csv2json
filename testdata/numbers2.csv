binary,octal,decimal,hex,float
0b0110,0o1274,75,0x0000FF,10.25
