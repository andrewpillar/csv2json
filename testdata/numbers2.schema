binary  int  0
octal   int  0
decimal int  0
hex     int  0
