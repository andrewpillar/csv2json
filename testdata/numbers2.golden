{"binary":6,"decimal":75,"float":10.25,"hex":255,"octal":700}
